HTML/Answers/07.cookies-session-and-localstorage-storage-difference.md

```diff
@@ -0,0 +1,71 @@
+# Cookies
+
+- **Limited** storage space **4KB**(~4096B)
+- Only allow to store data as **strings**
+- Stored data is sent back to server on every **HTTP request** such as HTML, CSS, Images etc,
+- Can store only **20 cookies per domain**
+- In total **300** cookies are allowed on a site
+- Setting **HTTP only** flag will **prevent accessing cookies** via javascript
+- Can set **expiration** duration for auto deletion (can be set either from server or client)
+
+**Example**:
+
+```js
+// Set with expiration & path
+document.cookie = "name=Gokul; expires=Thu, 18 Dec 2016 12:00:00 UTC; path=/;";
+
+// Get
+document.cookie;
+
+// Delete by setting empty value and same path
+document.cookie = "name=; expires=Thu, 18 Dec 2016 12:00:00 UTC; path=/;";
+```
+
+# Session Storage
+
+- Storage space is **more than cookies** 5MB (~5120KB)
+- Storage space may **vary a little** based on the browser
+- Only allow to store data as **strings**
+- Data is available per **window or tab**
+- Once window or tab is closed stored data is **deleted**
+- Data will be **only available on same origin**
+
+**Example**:
+
+```js
+// Set
+sessionStorage.setItem("name", "gokul");
+
+// Get
+sessionStorage.getItem("name"); // gokul
+
+// Delete
+sessionStorage.removeItem("name"); 
+
+// Delete All
+sessionStorage.clear();
+```
+
+# Localstorage Storage
+
+- Storage space is similar to **session storage**
```

Comment:
it would be great if you mention the size here also instead of saying it's similar to session storage